File: Nivel2-Funciones-procedimientos-condicionales/mayor.c
#include <stdio.h>


int devolverMayor(int A, int B, int C) {
  if(A == B == C) {
    printf("** Los numeros son iguales **");
  }
  else {
    if(A > B && A > B) {
      printf("** El mayor es: %d **", A);
    }
    else {
      if(B > A && B > C) {
        printf("** El mayor es: %d **", B);
      }
      else {
        printf("** El mayor es: %d **", C);
      }
    }
  }
}

int main() {
  int A, B, C;

  printf("Ingrese el primer numero: ");
  scanf("%d", &A);
  printf("Ingrese el segundo numero: ");
  scanf("%d", &B);
  printf("Ingrese el tercer numero: ");
  scanf("%d", &C);

  devolverMayor(A, B, C);

}

File: Nivel2-Funciones-procedimientos-condicionales/mayor.py
#!/usr/bin/env python3


A = int(input("ingrese un numero\n"))
B = int(input("ingrese otro numero\n"))
C = int(input("ingrese un nuemero\n"))

if(A == B == C):
  print("Los numeros son iguales")
else:
  if(A > B and A > C):
    print("El mayor es " + str(A)
  else:
    if(B > A and B > C):
      print("El mayor es " + str(B))
    else:
      print("El mayor es " + str(C))
